python-lists-and-tuples/solution.py

```diff
@@ -0,0 +1,9 @@
+L = 'abcdefghijklmnopqrstuvwxyz'
+def lpass(alphabet):
```

Comment:
The first function should just return two lists. Then you want to combine those lists either in another function or just after the first function.